Gemfile

```diff
@@ -0,0 +1,66 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.1.2'
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem 'rails', '~> 7.0.4'
+
+# Use postgresql as the database for Active Record
+gem 'pg', '~> 1.1'
+
+# Use the Puma web server [https://github.com/puma/puma]
+gem 'puma', '~> 5.0'
+gem 'webrick'
+
+gem 'rubocop', '>= 1.0', '< 2.0'
+
+gem 'rack-cors'
+
+gem 'devise'
+
+gem 'devise-jwt'
+
+gem 'cancancan'
```

Comment:
- [ ] It seems that almost all the code related to `cancancan` has been either removed or commented out.


If not using `cancancan` anymore, please consider deleting the gem and all the related code from the corresponding files (like ability.rb).

Reply:
Issue resolved, thank you.